_data/terms-of-service.yml

```diff
@@ -477,14 +477,12 @@
 
     - title: 2. Included Features
       legalese: |
-        <p><strong>2.1. Reference Architecture Customization.</strong> To understand your architecture preferences, we will ask you to submit an online form that includes questions on how you desire to configure your infrastructure.</p>
-        <p><strong>2.2. Reference Architecture.</strong> Gruntwork will design, implement and deploy a standardized architecture for use with the cloud provider (e.g. Amazon Web Services) that you selected during registration (the "Reference Architecture") based on the requirements you specify via an online form, verbal discussion, or other mutually acceptable means.</p>
-        <p><strong>2.3. Reference Architecture.</strong> Gruntwork will design, implement and deploy a standardized architecture for use with the cloud provider (e.g. Amazon Web Services) that you selected during registration (the "Reference Architecture") based on the requirements you specify via an online form, verbal discussion, or other mutually acceptable means.</p>
-        <p><strong>2.4. Code in Client Git Repos.</strong> The code for the Reference Architecture will be committed to your Git repositories.</p>
+        <p><strong>2.1. Customization.</strong> To understand your architecture preferences, we will ask you to submit a configuration file that includes options on how to customize your infrastructure. We cannot accommodate any customization requests beyond the pre-defined options.</p>
+        <p><strong>2.2. Deployment.</strong> Gruntwork will implement and deploy a standardized architecture for use with the cloud provider (e.g. Amazon Web Services) that you selected during registration (the "Reference Architecture") based on the requirements you specify via an online form, verbal discussion, or other mutually acceptable means. If you update the Reference Architecture code or change your configuration preferences while we are deploying, we may be unable to complete the deployment.</p>
+        <p><strong>2.3. Code Delivery.</strong> The code for the Reference Architecture will be committed to your Git repositories.</p>
```

Comment:
NOTE: Technically, we are no longer committing to the customers' git repositories, so we should probably change this.

Reply:
Changed in [1058352](https://github.com/gruntwork-io/gruntwork-io.github.io/commit/1058352e1869a1e11ab75b4b9cdceb798d81f376). Do you think that captures how we handle things?